expan/core/experiment.py

```diff
@@ -218,26 +216,32 @@ def outlier_filter(self, data, kpis, percentile=99.0, threshold_type='upper'):
                         "Denominator '{}' of the derived KPI does not exist in the data in outlier filtering.".format(kpi.denominator))
                 kpi.make_derived_kpi(data)
 
-        # check if provided percentile is valid
-        if 0.0 < percentile <= 100.0 is False:
-            raise ValueError("Percentile value needs to be between 0.0 and 100.0!")
-        # check if provided filtering kind is valid
-        if threshold_type not in ['upper', 'lower']:
-            raise ValueError("Threshold type needs to be either 'upper' or 'lower'!")
+        admittable_thresholds = set(['upper', 'lower', 'two-sided'])
+
+        thresholds = thresholds or {}
```

Comment:
Why `thresholds = thresholds or {}`?

`thresholds` is expected to be a `dict` anyway. I don't see what this line does

Reply:
`thresholds` has `None` as a default value. `a = a or []/{}/set()` is a common idiom to resolve `None` to the default empty container.

Reply:
I was about to say: _"Shouldn't the default be `{}` then?"_

but then I remembered that python default values can have weird lifetimes

OK, I'll resolve!